dashboard/app/models/paired_user_level.rb

```diff
@@ -60,4 +60,38 @@ def self.pairs_by_user(users)
         memo
       end
   end
+
+  # Returns the User object representing the driver. Note that this returns nil
+  # if the driver's user account or the driver's progress was deleted.
+  def driver
```

Comment:
I would kind of expect this to be defined as an association like `belongs_to :teacher, class_name: 'User', foreign_key: :user_id`

Reply:
Oops sorry, it would have to be through user_level.. let me try again